js/components/productList.js

```diff
@@ -1,4 +1,96 @@
-import fetchProducts from "../fetch/fetch.js";
+
+import fetchProducts, { fetchOneProduct } from "../fetch/fetch.js";
+import { deleteProduct } from "../fetch/deleteProduct.js";
+import { addProduct } from "../fetch/addProduct.js";
+import { editProduct } from "../fetch/editProduct.js";
+
+const state = {
+  activeProductID: "",
```

Comment:
Nitpick: Consider using `null` to represent nothing instead of an empty string, which is falsy, but sometimes might cause confusion with empty string value.